release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -1274,6 +1310,12 @@ With this release, replacement control plane nodes are assigned to the correct i
 
 * Operator Lifecycle Manager (OLM) manages a set of `CatalogSource` objects from which Operators can be searched for and installed from. These catalog sources are the default sources for this action and are managed by Red Hat. However, it was possible to change these default catalog sources in a way that the OLM system would not notice. Modifying a default catalog source in a way that rendered it inoperable could cause cascading issues through OLM that might prevent a user from installing new or upgrading existing Operators on their cluster. This bug fix updates the `catalog-operator` runtime, which manages the default catalog sources, to be made aware of other changes to the `CatalogSource` spec. As a result, when a change is made to a default catalog source, OLM detects the change and resets it back to default. (link:https://issues.redhat.com/browse/OCPBUGS-5466[*OCPBUGS-5466*])
 
+[discrete]
+[id="ocp-4-13-perf-addon-operator-bug-fixes"]
+==== Performance Addon Operator
+
+* Previously, the `hwlatdetect` tool that the Cloud-native Functions (CNF) tests image uses for running latency tests on a CNF-enabled {product-title} cluster was configured with a detection period of `10` seconds. This configuration when combined with the detection width configuration of `0.95` seconds increased the likelihood of `hwlatdetect` missing a latency spike, because the tool monitors a node about 9.5 percent of the time over the allocated detection period. With this update, the detection width is set to `1` second, so that the tool can now monitor nodes for about 95 percent of the time over the allocated detection period. The remaining 5 percent of monitoring time is left unallocated so that the kernel can perform system tasks. (link:https://issues.redhat.com/browse/OCPBUGS-12433[*OCPBUGS-12433*])
```

Comment:
I am having a hard time understanding this bullet. I think it could be the first sentence that is tripping me up. I am having a hard time suggesting a rewrite because I am not sure if I am messing with any of  the terminology.

Reply:
Percent should be the symbol % based on isg https://www.ibm.com/docs/en/ibm-style?topic=measurement-numbers#proportions

Reply:
It also might be worth checking but you have the time in backticks and I am not 100% sure on if they need to be or not.

Reply:
Yeah, these numerical-based notes are hard to write. I'll need to pull in a specialized writer as I'm not familiar with this component.

Reply:
Noun string is correct. See these [docs](https://docs.openshift.com/container-platform/4.12/scalability_and_performance/cnf-performing-platform-verification-latency-tests.html).